src/System.IO.FileSystem.DriveInfo/src/System/IO/DriveInfo.Windows.cs

```diff
@@ -152,35 +152,11 @@ public long TotalSize
 
         public static DriveInfo[] GetDrives()
```

Comment:
Note that the Unix implementation was so simple it seemed silly to call DriveInfoInternal (and keep wrapping/unwrapping )